pkgs/tools/inputmethods/ibus-engines/ibus-libthai/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, makeWrapper, pkg-config, ibus, gtk3, libthai }:
+
+stdenv.mkDerivation rec {
+  pname = "ibus-libthai";
+  version = "0.1.4";
+
+  src = fetchurl {
+    url = "https://linux.thai.net/pub/ThaiLinux/software/libthai/ibus-libthai-${version}.tar.xz";
+    sha256 = "0iam7308rxkx2xwaabc5wyj7vrxgd4cr95pvwrkm8fr9gh2xnwgv";
+  };
+
+  nativeBuildInputs = [ makeWrapper pkg-config ];
```

Comment:
Why is `makeWrapper` here and in the other expressions?

Reply:
I shouldn't package when I am tired. I am not sure.